src/mlx/warnings.py

```diff
@@ -216,8 +221,8 @@ def warnings_wrapper(args):
 
     # Read config file
     if args.configfile is not None:
-        checker_flags = args.sphinx or args.doxygen or args.junit or args.coverity or args.xmlrunner
-        warning_args = (args.maxwarnings != 0) or (args.minwarnings != 0) or (args.exact_warnings != 0)
+        checker_flags = args.sphinx or args.doxygen or args.junit or args.coverity or args.xmlrunner or args.robot
+        warning_args = args.maxwarnings or args.minwarnings or args.exact_warnings
```

Comment:
There was a reason for comparison, so that they were all boolean around `or` . Lets keep it that way.

```suggestion
        warning_args = (args.maxwarnings != 0) or (args.minwarnings != 0) or (args.exact_warnings != 0)
```

Reply:
A zero is falsy. The value of `warning_args` renains the same for every use case that we care about. Don't you agree that the removal of the comparisons improves readability?

Reply:
Codeclimate also complained when the comparisons were present.

Reply:
To me, the comparison to 0 produces a boolean True/False, but that is from C, so for me readability was much more explicit before. Now you assume that 0 evaluates to False before doing a boolean OR comparison. I also thought codeclimate thinks the same. Since it doesn't, lets resolve.